tests/components/unifiprotect/test_repairs.py

```diff
@@ -282,3 +282,31 @@ async def test_rtsp_no_fix_if_third_party(
 
     assert msg["success"]
     assert not msg["result"]["issues"]
+
+
+async def test_rtsp_no_fix_if_globally_disabled(
+    hass: HomeAssistant,
+    ufp: MockUFPFixture,
+    doorbell: Camera,
+    hass_ws_client: WebSocketGenerator,
+) -> None:
+    """Test no RTSP disabled warning if RTSP is globally disabled on integration."""
+
+    for channel in doorbell.channels:
+        channel.is_rtsp_enabled = False
+
+    # Set RTSP globally disabled in config entry options
+    hass.config_entries.async_update_entry(
+        ufp.entry,
+        options={**ufp.entry.options, CONF_DISABLE_RTSP: True},
+    )
+
+    await init_entry(hass, ufp, [doorbell])
+    await async_process_repairs_platforms(hass)
+    ws_client = await hass_ws_client(hass)
+
+    await ws_client.send_json({"id": 1, "type": "repairs/list_issues"})
+    msg = await ws_client.receive_json()
+
+    assert msg["success"]
+    assert not msg["result"]["issues"]
```

Comment:
While I have seen many integrations doing the same, I think the bellow has the same output with a cleaner approach:
https://github.com/home-assistant/core/blob/708b3dc8b237caea7648e359d41a19979bfce486/tests/components/shelly/test_repairs.py#L339